src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String RESULT = "result";
+    public static final int NAME_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+    public static final String DELIMITER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> readDataList = readDataFromFile(fromFileName);
+        String dataToWrite = valuesCounter(readDataList);
```

Comment:
please try to create more specific name and rename method. the `valuesCounter` not is not explain clearly what the method do.